source/_integrations/lcn.markdown

```diff
@@ -66,7 +66,73 @@ The configuration of the `lcn` devices and entities is completely done using a w
+
+The **Modules / Groups** tab provides an overview of your configured LCN modules and groups, showing their name, ID, and segment ID. The LCN Configuration Panel attempts to derive names from the LCN modules; if a module has no name or is part of a group, a standard name is assigned.
+
+#### Scanning modules
+
+To initiate a scan for LCN modules on the bus, in the top-right corner, select the three dots {% icon "mdi:dots-vertical" %} menu and select **Scan Modules**.
+- **Result**: This process polls each module for its name and serial number. 
+- When all module responses have been received, they will be listed in the device list.
+- Scanning modules may take several seconds. The pop-up dialog will close automatically once the process is complete.
+
+#### Adding devices
+
+If module scanning fails or a module is unavailable on the bus, you can manually add it. Groups can also be created manually.
+
+1. To add a module or group manually, select the **Create Module/Group** button to open the dialog.
+
+  ![Create module/group dialog](/images/integrations/lcn/lcn_create_device.png)
+
+2. Select whether you want to add a module or group and enter the desired `segment id` and module/group `id`. 
+3. To add the new device, select **Create**.
+
+#### Deleting devices
+
+To delete a single device, select the trash can icon next to it. 
+- **Result**: This will remove the device from the device list and from Home Assistant, including any associated entities.
+
+To delete multiple devices at once, enable selection mode.  Select the desired entries, then, in the top-right corner, select  **Delete Selected**.
+
+### Configuring entities
+
+Entities configured for all devices are listed on the **Entities** tab. 
+
+- To view entities for a specific device (module or group), select the device entry in the **Modules / Groups** tab. This opens the **Entities** tab, filtering entities to show only those for the selected device. 
+- To apply custom filters, enable the filter option.
+
+  ![Create module/group dialog](/images/integrations/lcn/lcn_entities_page.png)
+
+#### Adding entities
+
+To create a new entity, select **Create Entity**, which opens the following dialog.
+
+![Create entity dialog](/images/integrations/lcn/lcn_create_entity.png)
+
+Select the module or group for which to create the entity from the dropdown menu. (If a single module or group filter is applied, it will be pre-selected.)
+
+Choose the domain (platform) for the entity and enter a name. You can change this name later within the Home Assistant entity settings.
+
+Depending on the selected domain, additional options will be shown. To add the entity to the list and to Home Assistant, enter the required information and select **Create** .
```

Comment:
```suggestion
1. To create a new entity, select **Create Entity**, which opens the following dialog.

  ![Create entity dialog](/images/integrations/lcn/lcn_create_entity.png)

2. Select the module or group for which to create the entity from the dropdown menu. 
  - If a single module or group filter is applied, it will be pre-selected.
3. Choose the domain (platform) for the entity and enter a name.
  - You can change this name later within the Home Assistant entity settings.
4. Depending on the selected domain, additional options will be shown. To add the entity to the list and to Home Assistant, enter the required information and select **Create** .
```